fe/fe-core/src/main/java/com/starrocks/sql/optimizer/Optimizer.java

```diff
@@ -329,6 +330,7 @@ private OptExpression rewriteAndValidatePlan(OptExpression tree, TaskContext roo
 
     private OptExpression pushDownAggregation(OptExpression tree, TaskContext rootTaskContext,
                                               ColumnRefSet requiredColumns) {
+        tree = new PushDownDistinctAggregateRule().rewrite(tree, rootTaskContext);
```

Comment:
I suggest put this line below the config? so we can disable aggregate push down by using session vars.

```
if (context.getSessionVariable().getCboPushDownAggregateMode() == -1) {
            return tree;
}

 tree = new PushDownDistinctAggregateRule().rewrite(tree, rootTaskContext);

```